### PR TITLE
Terminology update

### PR DESCRIPTION
**Pull request by UlrikeS91**
This was somehow oversee, but should actually reflect the terminologies in `controlledTerms/terminologies` (without `controlledTerms/terminologies/terminology` of course)

@lzehl, especially look at metaDataModelTyp.jsonld if the `"name": "metadata model type"` is ok or should be changed. Preferably also cross-check if I actually added all terminologies (except for `terminology`) ;)  

@olinux FYI

### CODE REVIEW

File: terminologies/terminology/metaDataModelType.jsonld
Comment (lzehl) [inline terminologies/terminology/metaDataModelType.jsonld]: (meta)data model type

Comment (lzehl) [reply terminologies/terminology/metaDataModelType.jsonld]: to explain: it covers both metadata model types (e.g. openMINDS) AND data model types (e.g. BIDS)

File: terminologies/terminology/softwareFeatureCategory.jsonld
Comment (lzehl) [inline terminologies/terminology/softwareFeatureCategory.jsonld]: softwareFeatureCategory should be removed (for all: schemas, terminologies, etc) was depricated

**Review by lzehl — CHANGES_REQUESTED** [review]
@UlrikeS91 thank you :slightly_smiling_face: it just needs some small changes.

**Comment by lzehl** [comment]
@UlrikeS91 and I've double checked: you've got all :wink:

File: terminologies/terminology/metaDataModelType.jsonld
Comment (UlrikeS91) [reply terminologies/terminology/metaDataModelType.jsonld]: That's what I thought but didn't trust myself :D

File: terminologies/terminology/softwareFeatureCategory.jsonld
Comment (UlrikeS91) [reply terminologies/terminology/softwareFeatureCategory.jsonld]: If it is depricated, why have the terms in the terminology-folder been updated by @jagru20 3 days ago? 

The schema for this has been deleted already and isn't used anywhere anymore. So, @lzehl is of course right. It is depricated. @jagru20, since you've done recent updates to this depricated terminology, do you need these terms to be moved to ("controlledTerms/terminologies/softwareFeature")? If you don't need them at all, can we delete the entire folder ("controlledTerms/terminologies/softwareFeatureCategory") with all its content?

**Comment by UlrikeS91** [comment]
Changed `"name": "metadata model  type"` to `"name": "(meta)data model  type"`, as requested. Deleted depricated "softwareFeatureCategory.jsonld" in "controlledTerms/terminologies/terminology". 

@lzehl, waiting for feedback from @jagru20 about what to do with "controlledTerms/terminologies/softwareFeatureCategory"-folder - delete or move terms to "controlledTerms/terminologies/softwareFeature".

File: terminologies/terminology/softwareFeatureCategory.jsonld
Comment (jagru20) [reply terminologies/terminology/softwareFeatureCategory.jsonld]: The only reason why I made changes (i.e., changing the case of name property to lower case) was that the folder was still existent in my fork of the project. It is in deed deprecated and can be deleted.

If someone reads this later and wonders; The decision not to use the softwareFeatureCategory was drawn in https://github.com/HumanBrainProject/openMINDS_core/issues/142

Comment (UlrikeS91) [reply terminologies/terminology/softwareFeatureCategory.jsonld]: Okay, then I'll delete the whole thing :)

**Comment by UlrikeS91** [comment]
Ready for second review. Deleted all remains from "softwareFeatureCategory" within this PR now.

**Review by lzehl — APPROVED** [review]

**Comment by lzehl** [comment]
@UlrikeS91 thanks a lot :+1:

**Pull request merged**
